10_nwis_pull/src/nwis_pull.R

```diff
@@ -0,0 +1,109 @@
+#### pull ####
+
+# prepare a plan for downloading (from NWIS) and posting (to GD) one data file
+# per state
+plan_nwis_pull <- function(partitions, service) {
+
+  folders <- list(
+    tmp='10_nwis_pull/tmp',
+    out='10_nwis_pull/out',
+    log='10_nwis_pull/log')
+  
+  # after all wanted data have been pulled, this function will be called but
+  # doesn't need to create anything much, so just return NULL
+  # isolate the partition info for just one task
+  partitions_name <- deparse(substitute(partitions))
+  partition <- scipiper::create_task_step(
+    step_name = 'partition',
+    target_name = function(task_name, step_name, ...) {
+      sprintf('%s_partition_%s', service, task_name)
+    },
+    command = function(task_name, ...) {
+      sprintf("filter_partitions(partitions = %s, I('%s'))", partitions_name, task_name)
+    }
+  )
+
+  # download from NWIS, save, and create .ind file promising the download is
+  # complete, with extension .tind ('temporary indicator') because we don't want
+  # scipiper to make a build/status file for it
```

Comment:
neat trick!

Reply:
Don't think we use this anymore, so deleted .tind comments.